app/views/dashboard/index.slim

```diff
@@ -0,0 +1,4 @@
+- if can? :manage, Episode
+  h1 Admin dashboard
+  = link_to 'Create Podcast', new_episode_path, class: 'btn btn-outline-success my-2 my-sm-0'
```

Comment:
```
btn.btn.btn-outline-success.my-2.my-sm-0
  = link_to ...
```
Btw `my-2 my-sm-0` bad names for classes

Reply:
It is bootstrap classes

Reply:
So?  You can't apply it as all other classes? Only with `class:`?